cypress/page-objects/shopPage.js

```diff
@@ -0,0 +1,34 @@
+import BasePage from './basePage';
+import Chance from 'Chance';
+
+const chance = new Chance();
+
+class ShopPage extends BasePage {
+  selectCategory(allCategories) {
+    return chance.pickone(allCategories.collections);
+  }
+
+  selectProduct(selectedCategory) {
+    const selectedItem = chance.pickone(selectedCategory.items);
+    selectedItem.linkUrl = selectedCategory.linkUrl;
+    return selectedItem;
+  }
+
+  pickRandomProduct(listOfProducts) {
+    const selectedCategory = this.selectCategory(listOfProducts);
+    return this.selectProduct(selectedCategory);
+  }
```

Comment:
for all these methods please create `utils` / `service` / use `support` provided by Cypress folder and move them there.
these support methods seem reusable and do not relate to specific page

Reply:
done